plugins/kotlin-as-java/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+publishing {
+    publications {
+        register<MavenPublication>("kotlin-as-java-plugin") {
+            artifactId = "kotlin-as-java-plugin"
+            from(components["java"])
+        }
+    }
+}
+
+val kotlin_version: String by project
+dependencies {
+    implementation(kotlin("stdlib-jdk8"))
+    compileOnly(project(":coreDependencies", configuration = "shadow"))
+    testCompileOnly(project(":core"))
+    testCompileOnly(project(":coreDependencies", configuration = "shadow"))
+    testImplementation("junit:junit:4.12")
```

Comment:
These are probably redundant. `stdlib` is compileonly in parent project, `coreDependencies` are included in `core`, which is compileOnly in parent project, testCompileOnly `core` is included in testAPI, so is `junit`. TestAPI is included in the parent project

Reply:
I've removed some of them, but for some reason the ones I've left are required for it to build.